intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -238,6 +238,48 @@ \subsection{When will my job start?}
 It might be beneficial to request less resources (e.g. not requesting all cores
 in a workernode), since the scheduler often finds a ``gap'' to fit the job into
 more easily.
+
+\fi
+
+\ifgent
+
+\subsection{Specifying the cluster on which to run}
+\label{subsec:specifying-the-cluster-on-which-to-run}
+
+To use other clusters, you can swap the \verb|cluster| module.
+This is a special module that change what modules are available for you,
+and what cluster your jobs will be queued in.
+
+By default you are working on \defaultcluster. To switch to, e.g., \othercluster you need
+to redefine the environment so you get access to all modules installed on
+the \othercluster cluster, and to be able to submit jobs to the \othercluster scheduler
+so your jobs will start on \othercluster instead of the default \defaultcluster cluster.
+
+\begin{prompt}
+%\shellcmd{module swap cluster/\othercluster}%
+\end{prompt}
+
+Note: the \othercluster modules may not work directly on the login nodes, because the
+login nodes do not have the same architecture as the \othercluster cluster, they
+have the same architecture as the \defaultcluster cluster however, so this is why
+by default software works on the login nodes. See \autoref{sec:running-software-incompatible-with-host}
+for why this is and how to fix this.
+
+To list the available cluster modules, you can use the \verb|module avail cluster/| command:
+\begin{prompt}
+%\shellcmd{module avail cluster/}%
+------------------------------------------------------------------------------------ /etc/modulefiles/vsc ------------------------------------------------------------------------------------
+   cluster/delcatty (S,L)    cluster/golett (S)    cluster/phanpy (S)    cluster/skitty (S)    cluster/swalot (S)    cluster/victini (S)
+
+  Where:
+   S:  Module is Sticky, requires --force to unload or purge
+   L:  Module is loaded
+
+If you need software that is not listed, request it via https://www.ugent.be/hpc/en/support/software-installation-request
+\end{prompt}
```

Comment:
Now that I see this output, we should probably clarify what a "sticky" module is:

```
As indicated in the output above, each \verb|cluster| module is a so-called sticky module, i.e. it will not be unloaded when \verb|module purge| (see relevant section) is used.
```